Packs/FeedDShield/Integrations/FeedDShield/FeedDShield.py

```diff
@@ -0,0 +1,36 @@
+from CommonServerPython import *
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+
+    params['indicator_type'] = FeedIndicatorType.CIDR
+
+    params['url'] = 'https://www.dshield.org/block.txt'
+    params['ignore_regex'] = "[#S].*"
+    params['indicator'] = json.dumps({
+        "regex": r"^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\t[\d.]*\t(\d{1,2})",
+        "transform": "\\1/\\2"
+    })
+
+    fields = json.dumps({
+        "geocountry": {
+            "regex": "^.*\\t.*\\t[0-9]+\\t[0-9]+\\t[^\\t]+\\t([A-Z]+)",
+            "transform": "\\1"
+        },
+        "registrarabuseemail": {
+            "regex": "^.*\\t.*\\t[0-9]+\\t[0-9]+\\t[^\\t]+\\t[A-Z]+\\t(\\S+)",
+            "transform": "\\1"
+        }
+    })
+    params['fields'] = fields
+
```

Comment:
You haven't added the `custom_fields_mapping` parameter to `params` that you added to the httpapimodule in your other PR.

Reply:
once the other PR will be merged I'll add this.

Reply:
lets add this now

Reply:
added